Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -258,17 +263,55 @@ class SmartPage_Context
     }
   };
 
-  public SmartPage_PageRequestManager_endRequest = function (sender: unknown, args: Sys.WebForms.EndRequestEventArgs)
+  public SmartPage_PageRequestManager_endRequest = (sender: unknown, args: Sys.WebForms.EndRequestEventArgs) =>
```

Comment:
when you trigger the postback and there's no status code in the text field, an error response is sent but not displayed. that's strange because while it's not intended to send a blank field in the test, the handling on the client should still work. please test and check.